workbench-core/authorization/src/dynamicAuthorization/wbcGroupManagementPlugin.ts

```diff
@@ -74,7 +74,9 @@ export class WBCGroupManagementPlugin implements GroupManagementPlugin {
       return { data: { groupId } };
     } catch (error) {
       if (isRoleAlreadyExistsError(error)) {
-        throw new GroupAlreadyExistsError(error.message);
+        //Do not throw error, group can already exist but not exist in Dynamic Authz
```

Comment:
Is this an assumption we're making because of the error, or can a role not exist without a group?

Reply:
a role can exist in cognito but not exist in dynamic authz, it requires a status in order to exist in dynamic authz. After further thought, this should be reverted. I think it is better to keep those roles clean from being imported

Reply:
I agree, lets keep dynamicAuthZ groups and non-dynamicAuthZ groups separate.